src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Footer = props => {
```

Comment:
This looks great! Great to see you added the key to each element as you iterated through the array.